app/controllers/courses_controller.rb

```diff
@@ -3,49 +3,52 @@ class CoursesController < ApplicationController
   before_action :authenticate_user!
 
   def new_course
-    @page_title = 'Dashboard -> Add a course'
-    @course = Course.new({})
+    @page_title = I18n.t('controller.courses.new_course.add_new_course')
+    @course = Course.new
     authorize @course
   end
 
   def create_course
     @user = current_user
     @course = @user.courses.new(course_params)
     authorize @course if @course.present?
-    @enrollment = @course.enrollments.build(user: @user, enrollment_type: "instructor")
+    @enrollment = @course.enrollments.build(user: @user, enrollment_type: 'instructor')
     if @course.save && @enrollment.save
-      flash[:notice] = 'This course has been added'
+      flash[:notice] = I18n.t('controller.courses.create_course.course_added')
       redirect_to dashboard_show_course_path
     else
       render :new_course, status: :unprocessable_entity
     end
   end
 
   def show_course
-    @page_title = 'Dashboard -> Show courses'
+    @page_title = I18n.t('controller.courses.show_course.show_courses')
     @courses = policy_scope(Course)
     authorize @courses
   end
 
   def show_single_course
-    @page_title = 'Dashboard -> Show a courses'
+    @page_title = I18n.t('controller.courses.show_single_course.show_course')
     @course = Course.includes(:lessons).find_by(id: params[:id])
-    authorize @course
-    @lesson = Lesson.new({})
-    @content = Content.new({})
+    return invalid_params unless  @course.present?
+    authorize @course if @course.present?
```

Comment:
is the extra space after unless is intentional?
```suggestion
    return invalid_params unless @course.present?
    
    authorize @course if @course.present?
```